docs/index.md

```diff
@@ -0,0 +1,36 @@
+# DeviceManager
```

Comment:
This is almost the same file as README.md. I think that, as we keep documenting this component, they will naturally drift apart.